504.xml

```diff
@@ -52,123 +52,137 @@ https://www.omaspecworks.org/about/intellectual-property-rights/
 		<Mandatory>Optional</Mandatory>
 		<Resources>
```

Comment:
Resources 14 – 21 about band information are provided by the modem and are SIM related. They provide information that might be useful when ordering a profile, but not used by the (i/e)UICC or profile assistant. Considering the amount of resources in this object, is it useful to have these resources in a separate object? Regarding the description of the resource, how to put a list of bands in an integer? Where (e.g. what specification) do I find information what a specific integer mean?